demos/mqtt/mqtt_demo_plaintext/mqtt_demo_plaintext.c

```diff
@@ -635,58 +387,58 @@ static void eventCallback( MQTTContext_t * pContext,
         switch( pPacketInfo->type )
         {
             case MQTT_PACKET_TYPE_SUBACK:
-                LogInfo( ( "Subscribed to the topic %.*s.",
+                LogInfo( ( "Subscribed to the topic %.*s.\n\n",
                            MQTT_EXAMPLE_TOPIC_LENGTH,
                            MQTT_EXAMPLE_TOPIC ) );
                 /* Make sure ACK packet identifier matches with Request packet identifier. */
                 assert( globalSubscribePacketIdentifier == packetIdentifier );
                 break;
 
             case MQTT_PACKET_TYPE_UNSUBACK:
-                LogInfo( ( "Unsubscribed from the topic %.*s.",
+                LogInfo( ( "Unsubscribed from the topic %.*s.\n\n",
                            MQTT_EXAMPLE_TOPIC_LENGTH,
                            MQTT_EXAMPLE_TOPIC ) );
                 /* Make sure ACK packet identifier matches with Request packet identifier. */
                 assert( globalUnsubscribePacketIdentifier == packetIdentifier );
                 break;
 
             case MQTT_PACKET_TYPE_PINGRESP:
-                LogInfo( ( "PIGRESP received." ) );
+
+                /* Nothing to be done from application as library handles
+                 * PINGRESP. */
+                LogError( ( "PINGRESP should not be handled by the application callback.\n\n" ) );
```

Comment:
Why is this logged as an error? Isn't receiving the ping-response from the library a valid use-case of the library?

Reply:
Ahh yes this was something Muneeb and I just addressed. It is because the `MQTT_ProcessLoop` will never ever invoke the app callback when a PINGRESP is received. However, I think LogWarn is more appropriate.

Reply:
If ping-response is not supposed to be notified to the application, then we shouldn't show in the demos but instead include it within the `default` case of packet types. 
Also, the MQTT integration test should be updated to treat the packet type in the test event callback as an error with an assert failure (if not already changed)

Reply:
PINGRESP isn't notified for this demo because it uses MQTT_ProcessLoop, which handles keep alive. It shouldn't fail the integration tests since it's likely we want to test MQTT_ReceiveLoop as well